README.md

```diff
@@ -70,34 +70,34 @@ See also [About Us](http://px4.io/about-us/#development_team) (px4.io) and the [
 ## Supported Hardware
 
 This repository contains code supporting these boards:
-  * [Snapdragon Flight](https://docs.px4.io/en/flight_controller/snapdragon_flight.html)
-  * [Intel Aero](https://docs.px4.io/en/flight_controller/intel_aero.html)
-  * [Raspberry PI with Navio 2](https://docs.px4.io/en/flight_controller/raspberry_pi_navio2.html)
+  * [Snapdragon Flight](https://docs.px4.io/master/en/flight_controller/snapdragon_flight.html)
+  * [Intel Aero](https://docs.px4.io/master/en/flight_controller/intel_aero.html)
+  * [Raspberry PI with Navio 2](https://docs.px4.io/master/en/flight_controller/raspberry_pi_navio2.html)
   * [Parrot Bebop 2](https://dev.px4.io/en/advanced/parrot_bebop.html)
```

Comment:
Parrot Bebop 2 url link should be http://docs.px4.io/master/en/complete_vehicles/bebop.html

Reply:
Thanks! Updated!